your-firsts/README.md

```diff
@@ -0,0 +1,46 @@
+# My Firsts
+
+This project contains examples to perform your first transactions,
+create your first fungible and non-fungible tokens, etc.
+
+## Install
+
+```
+npm install
+```
+
+## Start a local devnet for testing and development
+
+Please refer to the documentation here: https://docs.alephium.org/full-node/getting-started#devnet
+
+## Compile
+
+Compile the TypeScript files into JavaScript:
```

Comment:
Should it be `Compile the contract files`?

Reply:
Nice catch, thanks!